client/core/rs/src/api/write/log_recording.rs

```diff
@@ -0,0 +1,74 @@
+use crate::entities::{
+  log_recording::LogRecording,
+  NoData, ResourceTarget,
+};
+use derive_empty_traits::EmptyTraits;
+use resolver_api::Resolve;
+use serde::{Deserialize, Serialize};
+use typeshare::typeshare;
+
+use super::KomodoWriteRequest;
+
+//=====================
+// StartLogRecording
+//=====================
+
+#[typeshare]
+#[derive(
+  Serialize, Deserialize, Debug, Clone, Resolve, EmptyTraits,
+)]
+#[empty_traits(KomodoWriteRequest)]
+#[response(StartLogRecordingResponse)]
+#[error(serror::Error)]
+pub struct StartLogRecording {
+  pub target: ResourceTarget,
+  /// Optional custom name for the recording
+  pub name: Option<String>,
+  /// Duration in days. None means until manually stopped.
+  pub duration_days: Option<f64>,
```

Comment:
```suggestion
  /// Unix timestamp in ms for when log recording expires.
  pub expire_ts: Option<I64>,
```
Should allow the client the flexibility to choose exact timestamp, for example 10 minutes vs 1 hr vs 2 days.

On frontend the most flexible is to use a calendar / time selector, but a simpler option is to use number input + dropdown. The dropdown would select from [Minutes, Hours, Days], and the input would specify the units.

For example when user select Minutes and input "10", will compute expire_ts = now() + 10 minutes, if user select Days and input "2", will pass now() + 2 days etc.